check-versions/.phpunit.result.cache

```diff
@@ -0,0 +1 @@
+C:37:"PHPUnit\Runner\DefaultTestResultCache":302:{a:2:{s:7:"defects";a:3:{s:29:"VersionTest::testPHP73Version";i:3;s:29:"VersionTest::testPHP74Version";i:3;s:29:"VersionTest::testPHP80Version";i:3;}s:5:"times";a:3:{s:29:"VersionTest::testPHP73Version";d:0.105;s:29:"VersionTest::testPHP74Version";d:0.076;s:29:"VersionTest::testPHP80Version";d:0.075;}}}
```

Comment:
I think this file not require. Add this file to  .gitignore

Reply:
Thanks I'll be updating this shortly, just have to disable stackdriver for php8, as it still needs some work for it to be compatible. The other two issues will be updated with the next submit.